### ISSUE TITLE
Синтаксис для использования fbrflash его для работы в среде Windows

### ISSUE THREAD

**Issue by hassansagar78** [forth32/balong-fbtools#1]
Каким образом мы можем использовать его в cmd .
я подписали прошивки huawei e5573cs-322, с тем чтобы i wanna flash его в Быстро загружающаяся заводскому режимудля пульта ДУ имеет высокий уровень безопасности .также обновленного погрузчик не работает withit..любым другим способом для борьбы с ней.

**Comment by forth32** [comment]
Do not google translate text.

**Comment by hassansagar78** [comment]
oops

how can we use fbrflash.exe in cmd  ..?
what is its syntax.?
e-g (fbrflash -p4 xxx.bin ) what is proper procedure to put the modem directely in fastboot mode which have high security ?

**Comment by forth32** [comment]
fbrflash -p4 -m - view partition tabley -
fbrflash -p4 -fn - dump partition n.

-o - dump data+OOB
-y - dump partition in yaffs2 format

**Comment by HUAWEI-MODEM** [comment]
thanx forth32  for your reply

**Comment by hassansagar78** [comment]
in which mode this program would work??
download mode or some else ?

**Comment by hassansagar78** [comment]
In This Mode The Program Should Work?
Download mode ,emergency mode or some else Mode ?

**Comment by forth32** [comment]
Only in download mode.
For emergency (usb boot) mode, first use balong_usbdload program to boot modem from usb (load usbloader). Fater loading usbloader, modem will enter download mode.

**Comment by HUAWEI-MODEM** [comment]
can we use this comand (fbrflash -p4 -m ) after at^godload  ?

**Comment by forth32** [comment]
No. fbrflash is utility for fastboot operation - modem must be in fastboot mode (12d1:36dd).

**Comment by HUAWEI-MODEM** [comment]
how to enter normal modem to fastboot mode for example i have huawei e8372h modem  what is Proper procedure to the put the modem directely in the mode fastboot ?

**Comment by forth32** [comment]
1. Enable fastboot mode in nvram:

at^nvwrex=53521,0,64,3D E0 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 01 00 00 00 01 00 00 00 00 00 00 00

2. Call power switch procedure. From modem linux console, enter command:

ecall power_off_reboot_operation

Modem will reboot in fastboot mode. To exit mode, power cycle modem, or use faastboot reboot command.

**Comment by HUAWEI-MODEM** [comment]
microsoft user can this?
(Call power switch procedure. From modem linux console, enter command: ecall power_off_reboot_operation Modem will reboot in fastboot mode. To exit mode, power cycle modem, or use faastboot reboot command.)

**Comment by karim1630** [comment]
How to read flash fille in fastboot from modem b310s

**Comment by AhmedAmmarNegoudi** [comment]
use balong_usbdload program to boot modem from usb (load usbloader)
how to do this with instructions 
./balong_usbload ??
